File: asterisks/Asterisks.cpp
#include <iostream>
using namespace std;

using std::endl;
using std::cout;

void PrintAstr ( int n ) {

int numberInput;
cin >> numberInput; 
 
  for (int i = 0; i<numberInput; i++ ) {
     cout << '*';
}
      cout << endl;
      
 
 }

int numberInput;
int main()
{
while (true) { 
cout << "Enter number of asterisks: " << endl ;
 PrintAstr( numberInput );
cout << "Do you want to start again? yes/no " << endl ;
string stringInput ;
cin >> stringInput;
if ( stringInput == "no" ) {
break ;
}
}
}

File: asterisks/Asterisks.java

import java.util.*;

public class Asterisks {
  public static void main (String[] args) { 



Scanner input = new Scanner(System.in);
while (true) {
System.out.println("Enter a number of asterisks: ");
int num = input.nextInt();
for (int i = 0;  i < num; i++) {
    System.out.print("*");
} 
    System.out.println(); 
System.out.println("Do you want to start again? yes/no ");
String ans;
ans = input.nextLine();
System.out.println("answer "+ans );

if ( ans == "no") {
break;
 

    }
   }
 }
}
